insecure/cmd/build_helper1.sh

```diff
@@ -0,0 +1,12 @@
+# temporarily make insecure/ a non-module to allow Docker to use corrupt builders there
```

Comment:
can we name these to reflect what they're doing? maybe `override_mods.sh` and `restore_mods.sh` or something like that

Reply:
https://github.com/onflow/flow-go/pull/3585/commits/d4937ff24a66108377e78b1512ad8a8eb26e2771